linkis-computation-governance/linkis-engineconn/linkis-engineconn-executor/accessible-executor/src/main/scala/org/apache/linkis/engineconn/callback/service/EngineConnCallback.scala

```diff
@@ -36,15 +36,15 @@ abstract class AbstractEngineConnStartUpCallback() extends EngineConnCallback wi
     protocol match {
       case protocol: EngineConnStatusCallback =>
         if (protocol.getStatus().equals(NodeStatus.Failed)) {
-          logger.error(s"protocol will send to lm: ${protocol}")
+          logger.error(s"EngineConn Start Failed protocol will send to lm: ${protocol}")
         } else {
```

Comment:
lm => LM or LinkisManager  
for easy to understand

Reply:
yes